test/test_torch.py

```diff
@@ -17833,21 +17833,37 @@ def test_strided_mm_bmm(self, device, dtype):
         torch_fn = lambda x: torch.mm(x, x)  # noqa: E731
         self.compare_with_numpy(torch_fn, np_fn, sx[0])
 
-    @onlyCPU
-    @dtypes(*(torch.testing.get_all_complex_dtypes() + [torch.float, torch.double]))
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
```

Comment:
you don't need this skip, numpy is always present

Reply:
removed